x-pack/test/security_solution_api_integration/test_suites/lists_and_exception_lists/authentication/exceptions/common/trial_license_complete_tier/configs/serverless.config.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { createTestConfig } from '../../../../../../../config/serverless/config.base.essentials';
```

Comment:
This config is listed under a directory `trial_license_complete_tier` but it is using the serverless essentials configuration. I think one of these things is incorrect?

Reply:
Thanks! I fixed it up!